packages/models-library/src/models_library/api_schemas_webserver/resource_usage.py

```diff
@@ -44,6 +47,9 @@ class ServiceRunGet(
     started_at: datetime
     stopped_at: datetime | None
     service_run_status: ServiceRunStatus
+    # Cost in credits
+    credit_cost: Decimal | None
+    transaction_status: CreditTransactionStatus | None
 
```

Comment:
can you add a small example like

https://github.com/itisfoundation/osparc-simcore/blob/2415e7858d69b89a274da2c68711a9b76ee56e3a/packages/models-library/src/models_library/api_schemas_storage/storage_schemas.py#L75-L89


this is automatically tested